pkg/crc/machine/vfkit/driver_darwin.go

```diff
@@ -22,7 +22,11 @@ func CreateHost(machineConfig config.MachineConfig) *vfkit.Driver {
 	vfDriver.VfkitPath = filepath.Join(constants.BinDir(), VfkitCommand)
 
 	vfDriver.VirtioNet = machineConfig.NetworkMode == network.SystemNetworkingMode
+
 	vfDriver.VsockPath = constants.TapSocketPath
+	vfDriver.VsockDaemonPort = constants.DaemonVsockPort
```

Comment:
inconsistent use of `VsockDaemonPort` and `DaemonsockPort`

Reply:
Fixed.